russian/call-mom/look_out_from_window/look_out_from_window.md~

```diff
@@ -0,0 +1 @@
+Выглянув в окно, Вы видите свои зеркальное отражение в какой-то серебристой поверхности
```

Comment:
Please make sure you name all of your files with the extension `.md`.  Otherwise, the files will not display correctly on GitHub.